docs: remove links to dw answers (#1037)
watson-developer-cloud/node-sdk: README.md

## Questions

> If you are having difficulties using the APIs or have a question about the Watson services, please ask a question at [Stack Overflow](http://stackoverflow.com/questions/ask?tags=ibm-watson-cognitive).

Reviewer: ```suggestion
If you have issues with the APIs or have a question about the Watson services, see [Stack Overflow](https://stackoverflow.com/questions/tagged/ibm-watson+node.js).
```

Reviewer: possible updates: link to the answers, not just posting a question, so that people can search for already answered questions. Also, use the SDK framework or language tag as well.

Reviewer 2: Agreed
